src/main/java/org/cpsc304/bigdata/db/DatabaseInitializer.java

```diff
@@ -14,209 +14,392 @@
 public class DatabaseInitializer {
 
     private static final String[] DROP_TABLES = {
-            "ClinicalTrail_Treatment", "Disease_ClinicalTrail",
+            "ClinicalTrial_Treatment", "Disease_ClinicalTrial",
+            "ClinicalTrial", "Treatment", "Disease",
             "DiagnosedBy", "DiagnosticTest", "WorkOn",
-            "ClinicalTrial", "Treatment",
             "SuffersFrom",
             "MedicalRecord", "Patient",
             "Strain", "InfectiousOrganism",
-            "InfectiousDisease", "PhysiologicalDisease", "HereditaryDisease", "HPattern", "DeficiencyDisease", "Disease",
-            "Researcher", "Physician", "User_Info", "User_Dept"
+            "InfectiousDisease", "PhysiologicalDisease",
+            "Researcher", "Physician", "User_Info", "User_Dept",
+            "HereditaryDisease", "HPattern", "DeficiencyDisease"
 
     };
     private static final String[] CREATE_TABLES = {
             "CREATE TABLE User_Dept (" +
-                    "Speciality VARCHAR(20) PRIMARY KEY," +
-                    "Department VARCHAR(20)" +
+                    "Speciality VARCHAR2(20) PRIMARY KEY," +
+                    "Department VARCHAR2(20)" +
```

Comment:
What's `VARCHAR2`?

Reply:
It's the same as VARCHAR but somehow better and more recommended according to Oracle docs.